app/queries/cohort_analytics_query.rb

```diff
@@ -33,37 +33,37 @@ def patient_counts_by_period(period, prev_periods, from_time: Time.current)
   end
 
   def patient_counts(cohort_start, cohort_end, report_start, report_end)
-    registered_patients = registered(cohort_start, cohort_end)
-    followed_up_patients = followed_up(registered_patients, report_start, report_end)
+    cohort_patients = cohort_patients(cohort_start, cohort_end)
```

Comment:
To be honest, I think it's ok to keep the `registered` terminology, since we scope the patients outside this area. Within here, I think it makes sense to say "given a set of patients, these are the total registered patients, these are the controlled, uncontrolled, and no-BPs".

As we start propagating assigned facility everywhere, we'll no longer have to get confused by the word "registration". Not a big deal either way.

Reply:
Even within this context, the idea of pure `registrations` breaks down when we group by `assigned_facility`. I guess the `cohort_patients` method could be called `registrations`👍 . since after the grouping it becomes more like `assigned_and_registered`. Changed in 9740b1f.